persistence-modules/core-java-persistence/src/main/java/com/baeldung/jdbcmetadata/JdbcMetadataApplication.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.jdbcmetadata;
+
+import org.apache.log4j.Logger;
+
+import java.sql.SQLException;
+
+public class JdbcMetadataApplication {
+
+    private static final Logger LOG = Logger.getLogger(JdbcMetadataApplication.class);
+
+    public static void main(String[] args) {
+        DatabaseConfig databaseConfig = new DatabaseConfig();
+        databaseConfig.init();
+        try {
+            MetadataExtractor metadataExtractor = new MetadataExtractor(databaseConfig.getConnection());
+            metadataExtractor.extractTableInfo();
+            metadataExtractor.extractSystemTables();
+            metadataExtractor.extractViews();
+            String tableName = "CUSTOMER";
+            metadataExtractor.extractColumnInfo(tableName);
+            metadataExtractor.extractPrimaryKeys(tableName);
+            metadataExtractor.extractForeignKeys("CUST_ADDRESS");
+            metadataExtractor.extractDatabaseInfo();
+            metadataExtractor.extractUserName();
+            metadataExtractor.extractSupportedFeatures();
+        }catch (SQLException e){
```

Comment:
```suggestion
        } catch (SQLException e){
```

Reply:
@KevinGilmore  Handled all the review comments by using the recommended formatter.